autosportlabs/racecapture/settings/prefs.py

```
@@ -346,6 +365,19 @@ def set_pref(self, section, option, value):
         if value != current_value:
             self.dispatch('on_pref_change', section, option, value)
 
+    def set_pref_list(self, section, option, value):
+        """
+        Set a preference value by list
+        :param section the configuration section for the preference
+        :type string
+        :param option the option for the section
+        :type string
+        :param value the preference value to set
+        :type value list (list of strings)
+        """
+        self.set_pref(section, option, ','.join(value))
```

Updated to log and swallow the exception. Can change it based on discussion.